bin/wait.sh

```diff
@@ -23,8 +25,9 @@ do
         exit 1
     fi
 
-    nc -z ${HOST} ${PORT}
-    if [ $? -eq 0 ]
+    ERR=0
+    nc -v -z ${HOST} ${PORT} || ERR=1
+    if [ "${ERR}" == "0" ]
```

Comment:
This was tricky. We want `nc` to die and then retry this script, so we need to figure out when it died without having the non-zero exit code kill the script entirely. This was the best way I could figure out to do it.